ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -281,11 +282,17 @@ def _get_matching_session_infos(self, instrument_type_id: str) -> List[SessionIn
             if instrument_type_id and instrument_type_id == info.instrument_type_id
         ]
 
+    def _default_closing(self, session: TSession) -> ContextManager[TSession]:
```

Comment:
This should be a free function, not a method. That's why my code snippet didn't indent this function or give it a "self" parameter.

Reply:
On further thought, it would be better to move this function to _drivers/__init__.py, rename it to `closing_session`, and rename the other function to something like `closing_session_with_ts_code_module_support`.